.circleci/config.yml

```diff
@@ -338,6 +350,7 @@ workflows:
             - test-java-18
             - test-node-js-18
             - test-ruby-18
+            - test-typescript-18
```

Comment:
The sort order of these is inconsistent. Though really longer term I'd love to refactor this whole file to use Circle CI job parameters etc, so doesn't matter for now :-)